### PR TITLE
added globalsat  LT-20/LT-20P

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/globalsat/LT-20/meta.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "LT-20/LT-20P",
+  "version": "1.0.0",
+  "manufacturer": "GlobalSat",
+  "url": "https://www.globalsat.com.tw/en/product-275227/Solar-Powered-LoRa-GPS-Tracker-for-Cattle-Cow-Livestock-and-Farming-LT-20-LT-20P-Series.html",
+  "description": "LT-20 is a solar LoRaWAN tracker designed for range animals use. LT-20P is a solar LoRaWAN tracker designed for asset tracking purpose.",
```
Comment (mariusgiger) [inline types/globalsat/LT-20/meta.json:6]: what do you mean by: range animals use?

**Review by mariusgiger — DISMISSED** [review]
looks good, one comment

File: types/globalsat/LT-20/meta.json (hunk 1 above)
Comment (J2thatsme) [reply types/globalsat/LT-20/meta.json]: probably free range animals zb Cows etc. Changed it anyways

File: types/globalsat/LT-20/meta.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "url": "https://www.globalsat.com.tw/en/product-275227/Solar-Powered-LoRa-GPS-Tracker-for-Cattle-Cow-Livestock-and-Farming-LT-20-LT-20P-Series.html",
+  "description": "LT-20 is a solar LoRaWAN tracker designed for animal tracking. LT-20P is a solar LoRaWAN tracker designed for asset tracking purpose.",
+  "author": "Akenza AG",
+  "firmwareVersion": "V1.0.0",
+  "loraDeviceClass": "A",
+  "availableSensors": [
+    "GPS"
+  ],
+  "outputTopics": [
+    "default"
```
Comment (mariusgiger) [inline types/globalsat/LT-20/meta.json:14]: add all topics here

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**